dotnet/src/Connectors/Connectors.AI.OpenAI/Policies/NoRetryPolicy.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.Pipeline;
+
+namespace Microsoft.SemanticKernel.Connectors.AI.OpenAI.Policies;
+/// <summary>
+/// Represents a custom HTTP pipeline no retry policy.
+/// </summary>
+public class NoRetryPolicy : HttpPipelinePolicy
```

Comment:
Rather than create a custom policy for this, can uses just use `new RetryPolicy(maxRetries: 0)`?